CLUEstering/CLUEstering.py

```diff
@@ -215,36 +189,158 @@ class clusterer:
     """
 
     def __init__(self, dc_: float, rhoc_: float, outlier_: float, ppbin: int = 10):
-        try:
-            if float(dc_) != dc_:
-                raise ValueError('Error: wrong parameter type\nThe dc parameter must be a float.')
-            self.dc_ = dc_
-            if float(rhoc_) != rhoc_:
-                raise ValueError('Error: wrong parameter type\nThe rhoc parameter must be a float.')
-            self.rhoc = rhoc_
-            if float(outlier_) != outlier_:
-                raise ValueError("Error: wrong parameter type\n"
-                                 + "The outlier parameter must be a float.")
-            self.outlier = outlier_
-            if not isinstance(ppbin, (int)):
-                raise ValueError("Error: wrong parameter type\nThe ppbin parameter must be a int.")
-            self.ppbin = ppbin
-        except ValueError as ve_:
-            print(ve_)
-            sys.exit()
+        self.dc_ = dc_
+        self.rhoc = rhoc_
+        self.outlier = outlier_
+        self.ppbin = ppbin
 
         # Initialize attributes
         ## Data containers
-        self.clust_data = clustering_data()
+        self.clust_data = None
         self.scaler = StandardScaler()
```

Comment:
do we want to let the user decide the Scaler? or maybe not use the scaler if the data have been already normalized?

Reply:
I think that it's good to always scale,, also because it makes it easier to find the best combination of parameters.

Reply:
I was thinking of avoiding scaling if already scaled to save some time, specially in case of big datasets. I guess we can leave the StandardScaler as default, but maybe the user can specify to "not scale", what do you think?

Reply:
Ok makes sense. I'll add the option.